app/code/Magento/Eav/Setup/EavSetup.php

```diff
@@ -689,17 +689,17 @@ public function getAttributeGroupByCode($entityTypeId, $setId, $code, $field = n
      * @param int|string $entityTypeId
      * @param int|string $setId
      * @param int|string $groupId
-     * @return $this
+     * @return int
      * @throws LocalizedException
      */
     public function getAttributeGroupId($entityTypeId, $setId, $groupId)
     {
         if (!is_numeric($groupId)) {
-            $groupId = $this->getAttributeGroup($entityTypeId, $setId, $groupId, 'attribute_group_id');
+            $groupId = (int)$this->getAttributeGroup($entityTypeId, $setId, $groupId, 'attribute_group_id');
```

Comment:
A lot of integration tests were fallen. Loks like it caused that when **getAttributeGroup** returns **false** then after this change **$groupId** would be **0** because `(int) false === 0` then we don't go into next condition `if (!is_numeric($groupId)) {}` as was before and as result function **getAttributeGroupId** will return **0** instead of **default attribute group id**.